src/style.css

```diff
@@ -0,0 +1,53 @@
+
```

Comment:
Remove extra line (it's relevant only for start of file but we need have extra line in the end of file and it's ok)